eslint.config.mjs

```diff
@@ -0,0 +1,13 @@
+import globals from "globals";
```

Comment:
Could you please tell me why we need this file?

Reply:
this file shouldn't be here, will update in next PR